config/default/system.site.yml

```diff
@@ -8,7 +8,7 @@ slogan: ''
 page:
   403: ''
   404: ''
-  front: /node/399
+  front: /node/99
```

Comment:
Presumably config_ignore will... ignore this?

Reply:
Reverted